scripts/vectorfield.js

```diff
@@ -0,0 +1,82 @@
+// Create guides to visualize
+const guides = [
+    new Bezier({x: 45, y: 200}, {x: 120, y: 250}, {x: 120, y: 80}, {x: 250, y: 100}),
+    new Bezier({x: 300, y: 300}, {x: 500, y: 100}, {x: 300, y: 20}, {x: 520, y: 50})
+];
+
+const SCALE = 2;
+
+const canvas = document.createElement('canvas');
+const width = 550;
+const height = 400;
+canvas.width = width * SCALE;
+canvas.height = height * SCALE;
+document.body.appendChild(canvas);
+
+const ctx = canvas.getContext('2d');
+ctx.lineCap = 'round';
+
+const drawGuides = () => {
+    ctx.lineWidth = 4 * SCALE;
+    ctx.strokeStyle = '#F0F';
```

Comment:
Could we use a calder colour object here, and then pass in the string representation?

Reply:
Ahh nevermind, for some reason I thought we had an `asString` method for the Colour interface. I was wrong